spec/lib/clearbit/discovery_spec.rb

```diff
@@ -0,0 +1,20 @@
+require 'spec_helper'
+
+describe Clearbit::Discovery do
+  before do |example|
+    Clearbit.key = 'clearbit_key'
+  end
+
+  context 'Discovery API' do
```

Comment:
I'd remove the `context` here and just use the `it`. The context should be used for adding meta-data around the state of system:

```ruby
describe "launch the rocket" do
  context "all ready" do
  end
 
  context "not ready" do
  end
end
```